cats/cats-sql/src/test/groovy/com/netflix/spinnaker/cats/sql/SqlCacheSpec.groovy

```diff
@@ -38,7 +40,7 @@ class SqlCacheSpec extends WriteableCacheSpec {
     ((SqlCache) cache).merge('foo', data)
 
     then:
-    1 * ((SqlCache) cache).cacheMetrics.merge('test', 'foo', 1, 1, 0, 0, 2, 1, 0)
+    1 * ((SqlCache) cache).cacheMetrics.merge('test', 'foo', 1, 1, 0, 0, 1, 1, 0)
```

Comment:
H2 special handling resulted in an additional select statement, which is no longer the case.